.project-keeper.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  - type: maven
+    path: pom.xml
+    modules:
+      - jar_artifact
```

Comment:
```suggestion
```
No need to upload artifact to github release. We deploy to maven central only.

This means you can delete the release_droid_upload_github_release_assets.yml workflow.